interface/product_registration/product_registration_controller.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * ProductRegistrationController
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Matthew Vita <matthewvita48@gmail.com>
+ * @link    http://www.open-emr.org
+ */
+
```

Comment:
Place security flags here prior to including globals.php:
``````
$fake_register_globals=false;
$sanitize_all_escapes=true;
``````
And if need access to this without user authentication, then use the following prior to including globals.php:
``````
$ignoreAuth=true;
``````